x/crisis/types/authority.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type CrisisAuthority sdk.AccAddress
```

Comment:
Why is there a need for a dedicated type (alias)?

Reply:
I thought it would be the clearest. If we named them all `Authority` there would be name collision.

What other thing we could have done is having a `map[string[sdk.AccAddress]` as input for all the modules (for the authority) where the key is the module key. Then we can just check if the module name is present. 

Would you prefer that?

Reply:
I've simplified it in d623377. What do you think?